workflows/podtato-head/workflow.yaml

```diff
@@ -0,0 +1,116 @@
+      steps:
+        - - name: install-application
+            template: install-application
+        - - name: install-chaos-experiments
+            template: install-chaos-experiments
+        - - name: pod-delete
+            template: pod-delete
+        - - name: revert-chaos
+            template: revert-chaos
+          #- name: delete-application
+          #  template: delete-application
+
+    - name: install-application
+      container:
+        image: litmuschaos/litmus-app-deployer:latest
+        args: ["-namespace={{workflow.parameters.adminModeNamespace}}","-typeName=resilient","-operation=apply","-timeout=400", "-label=helloservice"]
+
+    - name: install-chaos-experiments
+      container:
+        image: alpine/k8s:1.18.2
+        command: [sh, -c]
+        args:
+          - "kubectl apply -f https://hub.litmuschaos.io/api/chaos/master?file=charts/generic/experiments.yaml -n
+            {{workflow.parameters.adminModeNamespace}} ; sleep 30"
+
+    - name: pod-delete
+      inputs:
+        artifacts:
+          - name: pod-delete
+            path: /tmp/chaosengine.yaml
+            raw:
+              data: |
+                apiVersion: litmuschaos.io/v1alpha1
+                kind: ChaosEngine
+                metadata:
+                  name: podtato-head-pod-delete-chaos
+                  namespace: {{workflow.parameters.adminModeNamespace}}
+                  labels:
+                    litmuschaos.io/engine-context: lab
+                spec:
+                  appinfo:
+                    appns: {{workflow.parameters.adminModeNamespace}}
+                    applabel: 'name=podtato-head'
+                    appkind: 'deployment'
+                  annotationCheck: 'false'
+                  engineState: 'active'
+                  chaosServiceAccount: litmus-admin
+                  monitoring: true
+                  jobCleanUpPolicy: 'retain'
+                  components:
+                    runner:
+                      imagePullPolicy: Always
+                  experiments:
+                    - name: pod-delete
+                      spec:
+                        probe:
+                        - name: "check-podtato-head-access-url"
+                          type: "httpProbe"
+                          httpProbe/inputs:
+                            url: "http://podtato-head.{{workflow.parameters.adminModeNamespace}}.svc.cluster.local:9000"
+                            insecureSkipVerify: false
+                            responseTimeout: 100
+                            method:
+                              get:
+                                criteria: "=="
+                                responseCode: "200"
+                          mode: "Continuous"
+                          runProperties:
+                            probeTimeout: 1
+                            interval: 1
+                            retry: 1
+                            probePollingInterval: 0
```

Comment:
@ispeakc0de how do we treat a '0' s polling interval?

Reply:
With ```probePollingInterval: 0``` , It will keep on sending requests without any break to endpoint.
For podtato-head I found even if value is 1 it is enough to get  ready for other pod

Reply:
here 0 value means nothing(nil)

Reply:
https://github.com/litmuschaos/chaos-charts/pull/434/files#r608632678 will not be the case after the addition of readiness probe. It will take some time to restart(60s). hence probe will failed

Reply:
It is expected for the probe to fail for a single replica case and work for multiple